Detectors/Base/src/MaterialManager.cxx

```diff
@@ -27,6 +27,14 @@
 
 using namespace o2::Base;
 
+const std::vector<std::string> MaterialManager::mProcessIDToName = {
+  "PAIR", "COMP", "PHOT", "PFIS", "DRAY", "ANNI", "BREM", "HADR", "MUNU", "DCAY", "LOSS", "MULS", "CKOV"
+};
+
```

Comment:
how about using `unordered_map<EProc, std::string>`, so one has the direct link between enum value and string value. relying on the index is not type safe

Reply:
This makes it also more typesafe as well and I would agree.